app/views/static_pages/about.html.erb

```diff
@@ -0,0 +1,10 @@
+<% provide(:title, "About") %>
+<h1>About</h1>
+<p>
+  The <%= link_to "http://www.railstutorial.org/" do %>
+  <%= content_tag :em, "Ruby on Rails" %>
+  <% end %> is a
+  <%= link_to "book", "http://www.railstutorial.org/book" %> and
+  <%= link_to "screencast series", "http://screencasts.railstutorial.org/" %> to teach web development with
+  <%= link_to "Ruby on Rails", "http://rubyonrails.org/" %> This is the sample application for the tutorial.
```

Comment:
Please use I18n in all of the text for multi-language support. You can visit the following link to do this. 
[Rails Internationalization (I18n) API](http://guides.rubyonrails.org/i18n.html
)

Reply:
done